DataFormats/HGCalDigi/src/classes_def.xml

```diff
@@ -1,31 +1,31 @@
 <lcgdict>
-   <class name="HGCalElectronicsId" ClassVersion="3">
-   <version ClassVersion="3" checksum="2587023721"/>
-   </class>
 
-   <class name="HGCROCChannelDataFrame<HGCalDetId>" ClassVersion="3">
-   <version ClassVersion="3" checksum="2026905296"/>
-   </class>
+  <class name="HGCalElectronicsId" ClassVersion="3">
+  <version ClassVersion="3" checksum="2587023721"/>
+  </class>
 
-   <class name="HGCROCChannelDataFrame<HGCalElectronicsId>" ClassVersion="3">
-   <version ClassVersion="3" checksum="2054618736"/>
-   </class>
+  <class name="HGCROCChannelDataFrame<HGCalDetId>" ClassVersion="3">
+    <version ClassVersion="3" checksum="2026905296"/>
+  </class>
+  <class name="HGCROCChannelDataFrame<HGCalElectronicsId>" ClassVersion="3">
+    <version ClassVersion="3" checksum="2054618736"/>
+  </class>
```

Comment:
Not really part of this PR but something to note nevertheless. Class template instantiations should not include class version nor checksum
```suggestion
  <class name="HGCROCChannelDataFrame<HGCalDetId>"/>
  <class name="HGCROCChannelDataFrame<HGCalElectronicsId>"/>
```
Instead, the class template's `HGCROCChannelDataFrame<>` definition should contain `CMS_CLASS_VERSION(3)` (that needs `#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"`)

Reply:
Is it all right with you if we do this in a separate PR, so we do not have to touch `DataFormats/Common` here?

Reply:
Yes, separate PR is fine (in a way even better, since the issue was in the code already before this PR). But you wouldn't have to touch `DataFormats/Common`, but `DataFormats/HGCalDigi`, no?

Reply:
Ah, I misunderstood that I had to edit `DataFormats/Common/interface/CMS_CLASS_VERSION.h`. 

Where do I need to add `CMS_CLASS_VERSION(3)` exactly? After
`#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"` in [`DataFormats/HGCalDigi/interface/HGCROCChannelDataFrame.h`](https://github.com/CMS-HGCAL/cmssw/blob/dev/hackathon_base_CMSSW_14_1_X/DataFormats/HGCalDigi/interface/HGCROCChannelDataFrame.h)?

Reply:
Within the class template definition, see an example e.g. in
https://github.com/cms-sw/cmssw/blob/45e883d07884360b66a4958b1e727eed160439e1/DataFormats/Common/interface/StdArray.h#L232